Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp

```diff
@@ -882,3 +904,39 @@ gboolean webkit_website_data_manager_clear_finish(WebKitWebsiteDataManager* mana
 
     return g_task_propagate_boolean(G_TASK(result), error);
 }
+
+/**
+ * webkit_website_data_manager_get_local_storage_quota:
+ * @manager: a #WebKitWebsiteDataManager
+ *
+ * Get the #WebKitWebsiteDataManager:local-storage-quota property.
+ *
+ * Returns: Allowed maximum size of local storage in bytes
+ *
+ */
+guint32 webkit_website_data_manager_get_local_storage_quota(WebKitWebsiteDataManager *manager)
+{
+    g_return_val_if_fail(WEBKIT_IS_WEBSITE_DATA_MANAGER(manager), FALSE);
+    WebKit::WebsiteDataStore& datastore = webkitWebsiteDataManagerGetDataStore(manager);
+
+    return datastore.localStorageQuota();
+}
+
+/**
+ * webkit_website_data_manager_set_local_storage_quota:
+ * @manager: a #WebKitWebsiteDataManager
+ * @quota: quota to be set in bytes
+ *
+ * Get the #WebKitWebsiteDataManager:local-storage-quota property.
+ *
+ * Returns: Allowed maximum size of local storage in bytes
+ *
+ */
+void webkit_website_data_manager_set_local_storage_quota(WebKitWebsiteDataManager *manager, guint32 quota)
+{
+    g_return_if_fail(WEBKIT_IS_WEBSITE_DATA_MANAGER(manager));
+    WebKit::WebsiteDataStore& datastore = webkitWebsiteDataManagerGetDataStore(manager);
+
+    datastore.setLocalStorageQuota(quota);
+}
+
```

Comment:
These 2 methods are not necessary. You can have them if you want, but if you do, they must be private and not exposed to the API. The only way to interact with the property is when the object is constructed calling g_object_new.
Also, if you keep these methods, webkit_website_data_manager_get_local_storage_quota should be webkit_website_data_manager_local_storage_quota (without the get).

Reply:
Will remove the APIs as there is no intention to alter it once set on initialization